### ISSUE TITLE
Sharp drop in analytics with 1.7.8 release

### ISSUE THREAD

**Issue by kf6gpe** [flutter/flutter#37019]
We've seen a sharp drop in the analytics returned by 1.7.8. We need to either:
- root cause the issue and apply a fix, or
- understand the drop as incorrect analytics before.

**Comment by kf6gpe** [comment]
May be related to https://github.com/flutter/flutter/pull/34179.

**Comment by zanderso** [comment]
Fix for https://github.com/flutter/flutter/pull/34179 landed here https://github.com/flutter/flutter/pull/36208

A larger culprit is probably failing to send analytics when killed by a signal. The fix for that landed here: https://github.com/flutter/flutter/pull/36490

When these changes hit dev we should start getting some signal indicating whether these fixes are sufficient, or if there is some other problem.

**Comment by jonahwilliams** [comment]
We're currently waiting on a dev release to validate that the fixes are sufficient. If so, we can close the issue

**Comment by Hixie** [comment]
@jonahwilliams We have had a dev release now, what is the conclusion?

**Comment by zanderso** [comment]
Early signals indicate that this may be fixed. dev/1.8.3 is recording ~3x run commands per user over dev/1.8.2. I will close this, and we can open a new issue if the numbers don't totally recover.
